lib/fs.dart

```diff
@@ -230,10 +230,25 @@ class FS {
   /// Use the FSStatusListener mixin and override the onFSSession method to know when Fullstory is ready for events, logs, etc.
   ///
   /// For more information, see https://developer.fullstory.com/mobile/flutter/fullcapture/callbacks-and-delegates/
+  @Deprecated('Use addStatusListener instead to allow multiple delegates.')
   static void setStatusListener(FSStatusListener? listener) {
     FullstoryFlutterPlatform.instance.setStatusListener(listener);
   }
```

Comment:
Would it be better to remove this entirely? Or maybe leave it as deprecated for now, but note in the message and/or description that it will be removed in a future release?